README.md

```diff
@@ -322,6 +322,38 @@ $ oc scale dc/manageiq-orchestrator --replicas=1
 Under normal circumstances the entire first time deployment process should take around ~10 minutes, indication of issues can be seen
 by examination of the deployment events and pod logs.
 
+
+### Allowing manageiq images in kubernetes
+
+By default, kubernetes may not allow you to deploy images from `docker.io/manageiq`.  If so, deploying the operator may raise an error:
```

Comment:
I wouldn't say "By default", perhaps "Depending on your cluster's configuration" or something to that effect.

Reply:
Thanks, I reworded it.  Let me know what you think.